index.html

```diff
@@ -9,6 +9,19 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="container">
+    <div class="top-item">
+      <h1>Título del libro</h1><p>(género)</p>
+    </div>
```

Comment:
try to reduce the number of tags. I think that this `div` is unnecessary and `h1` tag is enough. Besides, use `span` tag instead of `p` tag.